app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -1,7 +1,7 @@
 module Hyrax
   class FileSetsController < ApplicationController
     include Hyrax::FileSetsControllerBehavior
-    include Hyrax::FileSetsControllerBehavior
+    include Hyrax::BreadcrumbsForCollections
```

Comment:
`BreadcrumbsForCollections` is for the CollectionsController (https://github.com/projecthydra-labs/hyrax/blob/5c2df85da6181a2ad18e3d23bc951699f8c3508f/app/controllers/hyrax/collections_controller.rb#L4) I don't think it's appropriate here.